packages/components/src/text-control/index.js

```diff
@@ -2,25 +2,50 @@
  * WordPress dependencies
  */
 import { useInstanceId } from '@wordpress/compose';
+import { forwardRef } from '@wordpress/element';
```

Comment:
We need to make sure this is listed in the package's references or we may run into problems:

https://github.com/WordPress/gutenberg/blob/7d1a260953643c93cb75e2c0ede31fa4da452ea2/packages/components/tsconfig.json#L7

We should add `{ "path": "../element" }` to that array.

Reply:
@sirreal Updated!